src/file.ts

```diff
@@ -3361,6 +3361,18 @@ class File extends ServiceObject<File> {
     });
   }
 
+  /**
```

Comment:
Should we beef up this documentation, maybe explaining why this might be useful? Also, it could use an `@example` to match the other doc blocks.

Reply:
@stephenplusplus Added example and updated doc